### PR TITLE
#72: added initial version of system_requirements.md

### LINKED ISSUE #72: Write system requirements and design for Kinesis Connector

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #72

### CODE REVIEW

File: docs/system_requirements.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# System Requirement Specification Exasol Kinesis Connector
+
+## Introduction 
+
+Exasol Kinesis Connector (short "EKC") is a plug-in for Exasol that provides the ability to transfer data from [Amazon Kinesis](https://aws.amazon.com/kinesis/) to the Exasol database.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
Exasol Kinesis Connector (short "EKC") is a User Defined Function (UDF) for Exasol that provides the ability to transfer data from [Amazon Kinesis](https://aws.amazon.com/kinesis/) to the Exasol database.
```

File: docs/system_requirements.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## About This Document
+
+### Target Audience
+
+The target audience are end-users, requirement engineers, software designers and quality assurance. See section ["Stakeholders"](#stakeholders) for more details.
+
+### Goal
+
+The EKC main goal is to provide a plug-in for transferring data from Amazon Kinesis directly to the Exasol database.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
The EKC main goal is to provide a UDF for transferring data from Amazon Kinesis directly to the Exasol database. 
```

File: docs/system_requirements.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+1. Reliability 
+1. Security
+1. Usability
+1. Performance efficiency
+
+## Stakeholders
+
+### Integrators
+
+Integrators integrate their solution with Exasol. They need to use Exasol as a consumer for data from Amazon Kinesis.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
Integrators integrate their solution with Exasol. They need to use Exasol as a consumer for the data from Amazon Kinesis.
```

File: docs/system_requirements.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+The following list gives you an overview of terms and abbreviations commonly used in OFT documents.
+
+* Data Consumer: a service retrieving data from Amazon Kinesis shards.
+* Data Record: is the unit of data stored in an Amazon Kinesis stream. 
+* Shard: an append-only log and a unit of streaming capability of Amazon Kinesis.
+
+## Features
+
+Features are the highest level requirements in this document that describe the main functionality of RLS.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
Features are the highest level requirements in this document that describe the main functionality of EKC.
```

File: docs/system_requirements.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+EKC connects to Kinesis Streams directly from Exasol.
+
+Covers:
+
+* [feat~transerring-data-from-kinesis-streams-to-exasol~1](#transferring-data-from-kinesis-streams-to-exasol)
+
+Needs: dsn
+
+### Transfer data to an Exasol Table
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
### Import data to an Exasol Table
```

File: docs/system_requirements.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+EKC transfers data from a Kinesis Stream directly to an Exasol table.
+
+Covers:
+
+* [feat~transerring-data-from-kinesis-streams-to-exasol~1](#transferring-data-from-kinesis-streams-to-exasol)
+
+Needs: dsn
+
+### Transfer without loosing data
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
### Import without loosing data
```

File: docs/system_requirements.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+`req~transfer-without-loosing-data~1`
+
+EKC transfers the data from a Kinesis Stream to an Exasol table without loosing records.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
EKC imports the data from a Kinesis Stream to an Exasol table without loosing records.
```

File: docs/system_requirements.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+Covers:
+
+* [feat~transerring-data-from-kinesis-streams-to-exasol~1](#transferring-data-from-kinesis-streams-to-exasol)
+
+Needs: dsn
+
+### Transfer data to an Exasol Table
+`req~transfer-data-to-an-exasol-table~1`
+
+EKC transfers data from a Kinesis Stream directly to an Exasol table.
```
Comment (morazow) [inline docs/system_requirements.md]:
```suggestion
EKC imports data from a Kinesis Stream directly to an Exasol table.
```

File: docs/system_requirements.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+### Transfer without loosing data
+`req~transfer-without-loosing-data~1`
+
+EKC transfers the data from a Kinesis Stream to an Exasol table without loosing records.
+
+Covers:
+
+* [feat~transerring-data-from-kinesis-streams-to-exasol~1](#transferring-data-from-kinesis-streams-to-exasol)
+
+Needs: dsn
```
Comment (morazow) [inline docs/system_requirements.md]: Another functional requirement should be to import data without duplicates.

Comment (AnastasiiaSergienko) [reply docs/system_requirements.md]: Added it

**Pull request merged**